### PR TITLE
Add contentful model and content exports

### PR DESCRIPTION
**Pull request by thinkybeast**
Adds Contentful files to import the content model and demo content into a user's space.

Includes a yarn command `yarn import:contentful:demo` as a shorthand for running the Contentful CLI command to import the demo to a given space.

### CODE REVIEW

File: gatsby-theme-landing-page/README.md [hunk 1]
```diff
@@ -38,9 +38,25 @@ CONTENTFUL_SPACE_ID="<SPACE_ID>"
 CONTENTFUL_DELIVERY_ACCESS_TOKEN="<ACCESS_TOKEN>"
 ```
 
-<!-- TODO: update these instructions once the JSON file is created -->
+Import the content model and demo data to your Contentful space
 
-Upload the `data/sample-data.json` to your Contentful space to set up the content model. Then, create your first landing page in Contentful.
+1. Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
+
+2. Add your space ID to `contentful/import-demo-config.json` and `contentful/import-model-only-config.json`
```
Comment (jxnblk) [inline gatsby-theme-landing-page/README.md:45]: I wonder if there's a way to leverage env vars for the contentful-cli (thinking about ways to reduce steps here) 🤔

File: gatsby-theme-landing-page/README.md [hunk 2]
```diff
@@ -38,9 +38,25 @@ CONTENTFUL_SPACE_ID="<SPACE_ID>"
 CONTENTFUL_DELIVERY_ACCESS_TOKEN="<ACCESS_TOKEN>"
 ```
 
-<!-- TODO: update these instructions once the JSON file is created -->
+Import the content model and demo data to your Contentful space
 
-Upload the `data/sample-data.json` to your Contentful space to set up the content model. Then, create your first landing page in Contentful.
+1. Install the [Contentful CLI (installation instructions)](https://www.contentful.com/developers/docs/tutorials/cli/installation/)
```
Comment (jxnblk) [inline gatsby-theme-landing-page/README.md:43]: If this is a single npm command, I'd say we should inline the instructions in a code block here, e.g. `npm i -g contentful-cli`

File: package.json [hunk 3]
```diff
@@ -11,6 +11,8 @@
     "start": "gatsby develop",
     "start:refresh": "ENABLE_GATSBY_REFRESH_ENDPOINT=true yarn start",
     "refresh": "curl -X POST http://localhost:8000/__refresh",
+    "import:contentful:demo": "contentful space import --config ./contentful/import-demo-config.json",
```
Comment (jxnblk) [inline package.json:14]: Did you try including contentful-cli as a dev dependency to avoid having to install the CLI globally?

**Review by jxnblk — APPROVED** [review]
Left some comments for discussion/follow-up, but LGTM. It also might make sense to add some sort of update command so that as we edit the content, we can automate updating the images and JSON here. 

Oh, and one last thought I forgot to comment on, since the `contentful` directory isn't required for the starter to work, should we name it `.contentful` similar to `.github` or `.circleci` directories?

File: gatsby-theme-landing-page/README.md (hunk 2 above)
Comment (thinkybeast) [reply gatsby-theme-landing-page/README.md:43]: yeah, there's even `npx contentful-cli <cmd>`

**Pull request merged**